apply/static/src/styles/apply.css

```diff
@@ -1,4 +1,4 @@
-/* TODO Remove once frontend jinja macros updated and we can use macros for headers - see apply-base.html */
-.header-pointer-events-none {
+/* Custom CSS for Apply */
+a.govuk-header__link--homepage {
     pointer-events: none;
   }
```

Comment:
Any idea why we have this custom code? Feels like we should just keep the normal behaviour unless there's a really good reason?

Reply:
It's so you don't get the underline hover behaviour on the gov.uk text and logo. It doesn't go anywhere so no point in making it look like a link